src/main/java/seedu/address/logic/commands/meeting/EditCommand.java

```diff
@@ -77,7 +77,16 @@ public CommandResult execute(ModelMeeting modelMeeting) throws CommandException
         }
 
         Meeting meetingToEdit = lastShownList.get(targetIndex.getZeroBased());
-        Meeting editedMeeting = createEditedMeeting(meetingToEdit, editMeetingDescriptor);
+        Meeting editedMeeting;
+
+        try {
+            editedMeeting = createEditedMeeting(meetingToEdit, editMeetingDescriptor);
+        } catch (IllegalArgumentException e) {
+            throw new CommandException(e.getMessage());
+        }
+
+        assert Meeting.isValidFromAndTo(editedMeeting.getFrom(), editedMeeting.getTo())
```

Comment:
> probably an exception? since its invalid user input not problems with our code

```
try {
            editedMeeting = createEditedMeeting(meetingToEdit, editMeetingDescriptor);
        } catch (IllegalArgumentException e) {
            throw new CommandException(e.getMessage());
        }

        assert Meeting.isValidFromAndTo(editedMeeting.getFrom(), editedMeeting.getTo())
                : "From should be earlier than To";
```
If you are referring to the assertion, it is to make sure that after the try-and-catch block, editedMeeting is a valid Meeting. If it's invalid, the catch block throw an Exception.